src/main/java/mekanism/common/base/IFluidHandlerWrapper.java

```diff
@@ -20,4 +20,12 @@
     FluidTankInfo[] getTankInfo(EnumFacing from);
 
     FluidTankInfo[] getAllTanks();
+
+    /**
+     * Called when accessing the handler from a null side. Overriding this allows replacing the local tanks with the
+     * networks tanks. Used by TileEntityMechanicalPipe to report the network's contents to TOP
+     */
+    default FluidTankInfo[] getAllNetworkTanks() {
```

Comment:
I'm not sure this needs to be a thing, shouldn't the Mechanical pipes be returning the network's buffer when connected anyway?

Reply:
Fixed them not exposing the full network's buffer. Originally thought this was intentional is why I did it differently. (Though I am keeping the read only null side handler I added to the gas network)